scripts/test/Muon/fitting_context_test.py

```diff
@@ -5,21 +5,153 @@
 #     & Institut Laue - Langevin
 # SPDX - License - Identifier: GPL - 3.0 +
 from __future__ import (absolute_import, unicode_literals)
+
+from collections import OrderedDict
 import unittest
-from mantid.py3compat import mock
-from Muon.GUI.Common.contexts.fitting_context import FittingContext, FitInformation
+
+from mantid.py3compat import iteritems, mock
+
+from Muon.GUI.Common.contexts.fitting_context import FittingContext, FitInformation, FitParameters
+
+
+def create_test_fit_parameters(test_parameters, global_parameters=None):
+    # needs to look like a standard fit table
+    fit_table = [{
+        'Name': name,
+        'Value': value,
+        'Error': error
+    } for name, (value, error) in iteritems(test_parameters)]
+
+    parameter_workspace = mock.MagicMock()
+    parameter_workspace.workspace.__iter__.return_value = fit_table
+    return FitParameters(parameter_workspace, global_parameters)
+
+
+class FitParametersTest(unittest.TestCase):
+    def test_equality_with_no_globals(self):
+        parameter_workspace = mock.MagicMock()
+        fit_params1 = FitParameters(parameter_workspace)
+        parameter_workspace = parameter_workspace
```

Comment:
is this line needed?

Reply:
No. That looks like a refactoring error.

Reply:
Its in a couple of places in this file

Reply:
I think I've caught these now.